### DISCUSSION TITLE
When running multiple persistent tasks, allow configuration for which to show first in `tui`

### DISCUSSION THREAD

**Discussion by jroru** [vercel/turborepo#9393] [Ideas]
### Goals

With a turbo monorepo containing multiple workspaces each with a persistent `start` task, I would like to ensure that `tui` shows the output from a specific package e.g. `web-app` as being "selected" in `tui` by default.

### Background

No alternative as I understand it. Only `turbo run start-special & turbo run start` which attempts to run two instances of `tui`.

### Proposal

Somehow allow configuration for task/workspace precedence in `tui`.

**Comment by anthonyshew** [comment]
Duplicate of https://github.com/vercel/turborepo/discussions/9146. The solution we're thinking we will go with is [here](https://github.com/vercel/turborepo/discussions/9146#discussioncomment-10759078).
